css/src/tokens/themes.scss

```diff
@@ -195,6 +201,9 @@ $themes: (
 		'body-background': $palette-black,
 		'body-background-medium': $palette-black,
 		'body-background-dark': $palette-black,
+		'alternate-background': $palette-grey-20,
```

Comment:
I think in `high-contrast` theme we should probably just keep it as black.

Reply:
Just for `alternate-background` or all three?

Reply:
all three

Reply:
Updated all three